src/app/components/test-card-detailed/test-card-detailed.component.ts

```diff
@@ -118,6 +125,16 @@ export class TestCardDetailedComponent {
       //this.imageTopActors[0] = this.base_image_URL1920 + resultt;
     })
 
+    this.dataService.getMovieExternalSourcesDetails(this.cardDetailMovieID).subscribe(result => {
+
+      this.movieExternalSourcesDetails = result.movieExternalSourcesDetailsFromDataService
```

Comment:
Please move it to separated private function

Reply:
done